### PR TITLE
Added Row and Column in Layout

### PR DESCRIPTION
**Pull request by wheregmis**
Added #1 
- [x] Use mangis instead of include_str macro
- [x] Add example for row and column

### CODE REVIEW

**Comment by wheregmis** [comment]
@DogeDark example works with include_str but doesnt work with manganis. I also saw an issue on collect asset repository. I will wait for it to be closed for this pr to make it work with manganis, or feel free to merge or review.

File: components/src/layout/column.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+/// ```
+
+pub fn Column(props: ColumnProps) -> Element {
+    rsx! {
+        // style { {include_str!("../../css-out/column.css")} }
+        div {
+            id: if let Some(id) = props.id { "{id}" },
+            class: if let Some(class) = props.class { "{class}" },
+            style: if let Some(style) = props.style { "{style}" },
+            style: "dxc-column",
```
Comment (DogeDark) [inline components/src/layout/column.rs:41]: I believe this should be ``class``

File: components/src/layout/column.rs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+///      id: "hi",
+///      style: "background-color: #f5f5f5; padding: 20px;",
+///      class: "bg-gray-100 p-5",
+///      Element
+///     }
+/// ```
+
+pub fn Column(props: ColumnProps) -> Element {
+    rsx! {
+        // style { {include_str!("../../css-out/column.css")} }
```
Comment (DogeDark) [inline components/src/layout/column.rs:36]: You can remove the ``style``s and ``include_str``s. At the moment I'm not worried about the assets not working for examples.

File: examples/preview/src/main.rs [hunk 3]
```diff
@@ -9,51 +8,28 @@ fn main() {
...
 
-        Accordion {
+        Row {
             id: "hi",
-            title: "Got Questions?",
-            AccordionItem {
-                title: "What is this?",
-                "This is just a preview of Dioxus Components! Pretty cool, right?"
+            style: "background-color: #f5f5f5; padding: 40px;",
+            class: "bg-gray-100 p-5",
```
Comment (DogeDark) [inline examples/preview/src/main.rs:25]: Looks like some Tailwind got through - I imagine it was for your testing?

**Review by DogeDark — CHANGES_REQUESTED** [review]
Looks pretty good! Just a few things.

**Comment by wheregmis** [comment]
Cleaned up 🎉

File: components/src/styles/column.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.dxc-row {
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
```
Comment (DogeDark) [inline components/src/styles/column.scss:6]: Here

File: components/src/styles/row.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.dxc-row {
+    padding: 10px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
```
Comment (DogeDark) [inline components/src/styles/row.scss:6]: Here

**Review by DogeDark — CHANGES_REQUESTED** [review]
One last thing. I tested it out and I had some problems with it displaying correctly with the ``align-items`` property. I removed them and all seems correct. Perhaps they should be removed?

**Comment by wheregmis** [comment]
> One last thing. I tested it out and I had some problems with it displaying correctly with the `align-items` property. I removed them and all seems correct. Perhaps they should be removed?

Sure, its removed ✔️

**Comment by DogeDark** [comment]
Awesome, this is great! Thanks for the addition!

**Pull request merged**